### PR TITLE
Admin: Combine org and admin user pages

### LINKED ISSUE #58828: Navigation IA: Combine the Org and Server Admin Users pages
#58211 will reorganize the Admin section. As part of that effort, we would like to combine the current /admin/users and /org/users pages into a single page/section.

Users who can see both /admin/users and /org/users should see the toggle, and /admin/users should be selected by default. Users who can see only one or the other should not see the toggle at all.

This change should sit behind the `topnav` feature toggle, so that it is enabled along with the rest of the nav updates.

Screenshot 2022-11-10 at 16 48 18

- [ ] TODO: @VesnaDean and @Clarity-89 meet to discuss feasibility of combining the pages using a toggle as shown in the image above.
- [ ] Make sure that the docs team is aware of these changes.

### PR DESCRIPTION
**Pull request by Clarity-89**
**What is this feature?**

Combine org and admin users pages into one page with a toggle for each section. 

**Why do we need this feature?**
To simplify the  admin pages structure. 

**Which issue(s) does this PR fix?**:

Fixes https://github.com/grafana/grafana/issues/58828

**Special notes for your reviewer**:

Screenshot 2022-11-28 at 10 20 46

### CODE REVIEW

File: public/app/core/components/NavBar/navBarItem-translations.ts [hunk 1]
```diff
@@ -184,7 +184,7 @@ export function getNavSubTitle(navId: string | undefined) {
     case 'serviceaccounts':
       return t('nav.service-accounts.subtitle', 'Use service accounts to run automated workloads in Grafana');
     case 'global-users':
-      return t('nav.global-users.subtitle', 'Manage and create users across the whole Grafana server');
+      return t('nav.global-users.subtitle', 'Manage users in Grafana');
```
Comment (joshhunt) [inline public/app/core/components/NavBar/navBarItem-translations.ts:187]: can you please run `yarn i18n:extract` after making this change 👍🏻

**Review by ashharrison90 — COMMENTED** [review]
couple of minor nits/questions, otherwise lgtm 🙌

File: public/app/routes/routes.tsx [hunk 2]
```diff
@@ -330,9 +332,7 @@ export function getAppRoutes(): RouteDescriptor[] {
     },
     {
       path: '/admin/users',
-      component: SafeDynamicImport(
-        () => import(/* webpackChunkName: "UserListAdminPage" */ 'app/features/admin/UserListAdminPage')
-      ),
+      component: config.featureToggles.topnav ? UserListPage : UserListAdminPage,
```
[the remark this answers is not in the document]
Comment (Clarity-89) [reply public/app/routes/routes.tsx]: Thought it didn't work earlier due to a conditional component, but turned out it only works for the default exports. Fixed now.

**Review by ashharrison90 — APPROVED** [review]
lgtm 🙌

**Pull request merged**